Services/PostService.cs

```diff
@@ -0,0 +1,53 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.EntityFrameworkCore;
+using Blogger.Data;
+
+namespace Blogger.Services
+{
+    public class PostService
+    {
+        public BloggerContext Context;
+        public PostRepository PostRepo;
+
+        public PostService(BloggerContext bloggerContext)
```

Comment:
Can we inject the repository here instead of the context?

Reply:
Sure, and I think this will maintain the layer chain like `Controller -> Service -> Repo`
I'll update  in next commit.